sklearn/tests/test_common.py

```diff
@@ -480,7 +480,7 @@ def test_estimators_do_not_raise_errors_in_init_or_set_params(Estimator):
     "IterativeImputer",
     "KBinsDiscretizer",
     "KNNImputer",
-    "KNeighborsTransformer",
+    # "KNeighborsTransformer",
```

Comment:
You need to remove this line

Reply:
I just updated the pull request. Please have a look